resume.json

```diff
@@ -0,0 +1,189 @@
+
+,"position":"Open Source Contributor [Contribution type]"
+
+,"location":{
+ "lat": "0",
+"long": "0"
+}
+
+,"url":"http://xyz.example.com"
+
+,"startDate":"2014-06-29"
+
+,"endDate":"2017-06-29"
+
+,"summary":"Give an overview of your responsibilities at the company"
+
+,"highlights":["Specify accomplishments and achievements"]}]
+ ,"publications":[{
+"name":"Deep learning and Artificial Intelligence"
+
+,"publisher":"XYZ, Computer Magazine"
+
+,"releaseDate":"2015-08-01 - [resume.json uses the ISO 8601 date standard]"
+
+,"resources":["Specify multiple resources with label"],"url":"http://www.computer.org.example.com/csdl/mags/co/2015/10/rx069-abs.html"
+
+,"summary":"Discussion of the advent of deep learning and artificial intelligence - Short summary of publication"
+
+}]
+ ,"legal":[{
+"name":"XYZ's patent on LZW compression, a fundamental part of the widely used GIF graphics format - [Add document name]"
+
+,"legalType":"Patent, Trademark, Copyright - Give the type of this document"
+
+,"description":"Give a brief description about this document"
+
+,"applicationDate":"2015-08-01"
+
+,"grantDate":"2016-09-01"
+
+,"endDate":"2020-09-03"
+
+,"resources":["Specify multiple resources with label"],"idNumber":"JP2004369746A - [Add the application number or Id Number]  "
+
+}]
+ ,"skills":[{
+"name":"Web Development"
+
+,"keyword":["List some keywords pertaining to the skill"]}]
+ ,"awards":[{
+"title":"Awarded Software Process Achievement Award "
+
+,"date":"2016-06-12"
+
+,"awarder":" IEEE"
+
+,"summary":"Received for my work in Deep learning and AI"
+
+}]
+ ,"projects":[{
+"name":"File Transfer application - [Name of the project]"
+
+,"location":{
+ "lat": "0",
+"long": "0"
+}
+
+,"description":"Developed a client and server based application - [Short summary of project]"
+
+,"highlights":["Specify multiple features"],"keywords":["Specify special elements involved"],"startDate":"2017-06-29"
+
+,"endDate":"2017-06-29"
+
+,"resources":["Specify multiple resources with label"],"url":""
+
+,"roles":["Specify your role on this project or in company"],"entity":"'greenpeace', 'corporationXYZ' - [Relevant company/entity affiliations]"
+
+,"type":"'volunteering', 'presentation', 'talk', 'application', 'conference'"
+
+}]
+ ,"certificates":[{
+"code":"1Z0-062"
+
+,"name":"XYZ Certified Application Specialist (MCAS) - [Add the certificate name]"
+
+,"website":""
+
+,"verification":""
+
+,"grantDate":"2017-06-29"
+
+,"score":"Exam result (PASS/FAIL, 100%, 100)"
+
+,"endDate":"2020-01-20"
+
+,"doesNotExpire":""
+
+}]
+ ,"references":[{
+"name":"Stephan Mark"
+
+,"company":"Xyz"
+
+,"position":"Senior Software Engineer"
+
+,"reference":"Joe blogs was a great employee, who turned up to work at least once a week. He exceeded my expectations when it came to doing nothing."
+
+}]
+ ,"languages":[{
+"language":"English, Spanish - [Name of language]"
+
+,"score":"20 - [Score for the language]"
+
+}]
+ ,"interests":[{
+"name":"Philosophy"
+
+}]
+ ,"meta":{
```

Comment:
Not all valid examples in meta and leftover descriptions.

Reply:
@stp-ip  That means to add them in the schema file first

Reply:
@stp-ip but I am not sure that needs to be entered by the user or our tool itself.

Reply:
Will write something up this week.

Reply:
@stp-ip Does format = uri also needs to be added on the canonical?

Reply:
The canonical should be a uri I would think. Or would anything count against that?

Reply:
yes along with a description example.